vsintegration/src/FSharp.Editor/QuickInfo/FSharpDeferredContent.fs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Microsoft.VisualStudio.FSharp.Editor
+
+open System.Windows
+open Microsoft.CodeAnalysis.Editor
+open Microsoft.CodeAnalysis.Editor.Shared.Utilities
+open Microsoft.CodeAnalysis
+open Microsoft.CodeAnalysis.Editor.Shared.Extensions
+open Microsoft.CodeAnalysis.Classification
+
+open CommonRoslynHelpers
+
+type internal FSharpDeferredContent(content: NavigableRoslynText seq, typemap: ClassificationTypeMap, navigateToRange) =
+
+    let formatMap = typemap.ClassificationFormatMapService.GetClassificationFormatMap("tooltip")
+
+    let props = 
+        ClassificationTags.GetClassificationTypeName
+        >> typemap.GetClassificationType
+        >> formatMap.GetTextProperties
+
+    let inlines = 
+      seq { 
+        for NavigableRoslynText(tag, text, xopt) in content do
+            let rangeOpt =
+                match xopt with
+                | Some xref ->
+                    match xref with
+                    | :? Microsoft.FSharp.Compiler.Range.range as range 
+                      when range.FileName <> "startup" ->
+                        Some range
+                    | _ -> None
+                | _ -> None
+            let inl = 
+                match rangeOpt with 
+                | Some range ->
+            
+                    let h = Documents.Hyperlink(Documents.Run(text))
+                    h.Click.Add <| fun _ ->
+                        Logging.Logging.logInfof "click! %A" range
```

Comment:
Remove logging, please. It will appear on VS output pane even if vsix is built in release configuration.